src/blssignature.cpp

```diff
@@ -80,6 +80,12 @@ BLSInsecureSignature BLSInsecureSignature::DivideBy(const BLSInsecureSignature&
     return result;
 }
 
+BLSInsecureSignature BLSInsecureSignature::Mul(const relic::bn_t n) const {
```

Comment:
We are using multiplicative notation, so multiplication should refer to multiplying two group elements, and exponentiation should refer to raising a group element to a scalar. So we should rename this to something like "Raise". 

I know this is not how relic does it, they use additive notation, but it's how it is described in the academic papers.